copier/errors.py

```diff
@@ -120,6 +123,36 @@ class MultipleYieldTagsError(CopierError):
     """Multiple yield tags are used in one path name, but it is not allowed."""
 
 
+class TaskError(CopierError):
+    """Exception raised when a task fails."""
+
+    def __init__(
+        self,
+        task_command: str | Sequence[str],
+        returncode: int,
+        stdout: str | bytes | None,
+        stderr: str | bytes | None,
+    ):
+        self.task_command = task_command
+        self.returncode = returncode
+        self.stdout = stdout
+        self.stderr = stderr
+        message = f"Task {task_command!r} returned non-zero exit status {returncode}."
```

Comment:
- Rename `task_command` to `command`, as it's implied by the exception name that the command is associated with a task.
- I'd surround the command in the message with backticks to improve readability when the command contains spaces.

```suggestion
        command: str | Sequence[str],
        returncode: int,
        stdout: str | bytes | None,
        stderr: str | bytes | None,
    ):
        self.command = command
        self.returncode = returncode
        self.stdout = stdout
        self.stderr = stderr
        message = f"Task `{command!r}` returned non-zero exit status {returncode}."
```

Reply:
the `!r` already add quotemarks to the output

Reply:
i will change the `.command` as suggested

Reply:
> the `!r` already add quotemarks to the output

Oh, I didn't know. 🤦 Good work then! 👏